### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by techionltd** [Pyhass/Hive-Custom-Component#130]
**Is your feature request related to a problem? Please describe.**
Family members setting the thermostat to unattainable temperatures (eg 29c) causes heating to run constantly, wasting energy.

**Describe the solution you'd like**
The ability to specify a maximum temperature that will override anything physically set at the thermostat.

**Describe alternatives you've considered**
None

**Additional context**
This graph illustrates a family member setting the temp way too high and the boiler 'topping out' at about 23c and running constantly

<img width="480" alt="Screenshot 2022-09-30 at 14 06 31" src="https://user-images.githubusercontent.com/3950050/193278076-432f0fc9-f022-4a73-aa58-8b53d4bac574.png">

**Comment by iaah05** [comment]
Wouldn't that better be handled by a Home Assistant Automation? Trigger would be if target is above 23 for X amount of time then set it back to your normal target.

**Comment by dougle03** [comment]
> Wouldn't that better be handled by a Home Assistant Automation? Trigger would be if target is above 23 for X amount of time then set it back to your normal target.

Indeed. I have a couple of nodeRed automations that handle temp control. Every 60 seconds it checks and then sets the correct temp for the time of day and occupancy. I have added a button that boosts the temp for 40 mins to .5c above the current temp. Tends to keep them happy...

**Comment by KJonline** [comment]
Don’t think this can be resolved by the integration any restrictions added in here would be overridden by the hive app or other integrations

I believe an automation would be the solution here
